client/src/components/LocationMap.tsx

```diff
@@ -268,9 +256,8 @@ export function EventsMapView(props: { posts: PostUserPreview[] }) {
       <Paper
         elevation={5}
         style={{
-          height: mapHeight,
-          width: mapWidth,
-          justifyContent: 'center',
+          height: '56vw',
```

Comment:
screenshot? how does it look now

Reply:
test resizing the window and different devices on university-board, it is currently deployed

Reply:
it is clipping on my screen